elser/challenges/default.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
That's good default but I think we can have multiple challenges for the same track in a follow up. This one focus on ELSER but we can have a different one that focus more on BM25 and maybe another one that perform the BM25/ELSER rank?

Reply:
So I think we should rename the track and call it msmarco-passage with different challenges.